relay-server/src/actors/global_config.rs

```diff
@@ -40,6 +40,23 @@ type UpstreamQueryResult =
 struct GetGlobalConfigResponse {
     #[serde(default)]
     global: Option<GlobalConfig>,
+    /// Instead of using [`Status`], we use StatusResponse as a separate field in order to not
+    /// make breaking changes to the api.
```

Comment:
I changed it to normal comment rather than doc comment, cause I think it's not directly intuitive why we do it like this, so its nice for reader to know it's because of API